### PR TITLE
initial import of .etch / preloading lib

### PR DESCRIPTION
**Pull request by aethernet**
Integrating .etch production into etcher-sdk

Change-type: minor

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,16 +54,19 @@
     "check-disk-space": "^2.1.0",
     "cyclic-32": "^1.1.0",
     "debug": "^3.1.0",
+    "docker-parse-image": "^3.0.1",
```
Comment (mcraa) [inline package.json:57]: is this used anywhere? I see only the own implementation imported.

File: lib/dotetch/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# .Etch
+
+## Genration
+
+Entry is `dotEtchStream.ts`
```
Comment (mcraa) [inline lib/dotetch/readme.md:5]: dotetch/index.ts ?

File: package.json (hunk 1 above)
Comment (aethernet) [reply package.json:57]: maybe not, that comes from the work or Rose, I'll have to double check

File: lib/dotetch/readme.md (hunk 2 above)
Comment (aethernet) [reply lib/dotetch/readme.md:5]: indeed that has been rewrote

File: package.json (hunk 1 above)
Comment (aethernet) [reply package.json:57]: yep I forgot but I actually rewrote the important parts of that library in TS 😅

File: lib/dotetch/readme.md (hunk 2 above)
Comment (aethernet) [reply lib/dotetch/readme.md:5]: Actually that readme was quite outdated, this is now fixed

**Comment by aethernet** [comment]
works continue here : https://github.com/balena-io-modules/etcher-sdk/pull/277

**Pull request closed without merging**